src/Umbraco.Web.BackOffice/Controllers/ContentController.cs

```diff
@@ -696,7 +695,7 @@ private async Task<ContentItemDisplay> PostSaveInternal(ContentItemSave contentI
                     // add the model state to the outgoing object and throw a validation message
                     var forDisplay = mapToDisplay(contentItem.PersistedContent);
                     forDisplay.Errors = ModelState.ToErrorDictionary();
-                    throw HttpResponseException.CreateValidationErrorResponse(forDisplay);
+                    return new ValidationErrorResult(forDisplay);
```

Comment:
You can't just replace this, you will need to handle the return type all the way to the top of the request. As you know, the throw statement skips the rest of the execution, the return don't.

In this case it can be handled by updating the return type of `PostSave` to `Task<ActionResult<ContentItemDisplay>>`